src/exceptions.ts

```diff
@@ -213,7 +213,7 @@ export class NotFoundException extends NodeRPCException {
 
 export class VaultNotFoundException extends NodeRPCException {
     constructor(message: string, causedBy?: Error) {
-        super(NodeRPCException.NOT_FOUND, NodeRPCException.IC_COLLECTION_NOT_FOUND, message, causedBy);
+        super(NodeRPCException.NOT_FOUND, NodeRPCException.IC_VAULT_NOT_FOUND, message, causedBy);
     }
```

Comment:
What's "IC" meaning in "IC_VAULT_NOT_FOUND"

Reply:
Internal Code, it will be removed later because of no need showing internal code to applications.